lms/config/__init__.py

```diff
@@ -48,6 +45,11 @@ def configure(settings=None):
     if env_settings['via_url'].endswith('/'):
         env_settings['via_url'] = env_settings['via_url'][:-1]
 
+    try:
+        env_settings["aes_secret"] = env_settings["aes_secret"].encode("ascii")[0:16]
```

Comment:
What if it's fewer than 16 chars? What happens?

Reply:
You'd end up with an aes_secret setting that's fewer than 16 chars and therefore less secure.  I think this aes_secret setting is used to encrypt the OAuth secrets that we receive from Canvas and store in our DB. Maybe we should add an additional startup check that it's at least 16 chars?

Reply:
Related: https://github.com/hypothesis/lms/issues/155